src/benchq/resource_estimation/openfermion_re.py

```diff
@@ -235,6 +236,8 @@ def get_physical_cost(
     num_t: int = 0,
     architecture_model: BasicArchitectureModel = BASIC_SC_ARCHITECTURE_MODEL,
     routing_overhead_proportion=0.5,
+    hardware_failure_tolerance=1e-3,
```

Comment:
Similar comment as in _footprint_analysis.py, just want to confirm that we want to reduce the default hardware failure tolerance to 1e-3.